MetaData/python/JobConfig.py

```diff
@@ -224,6 +225,13 @@ def parse(self):
             self.readProcessIdMap(self.options.processIdMap)
         
         if self.useAAA:
+            import subprocess
+            tleft = int(subprocess.check_output(["voms-proxy-info", "--actimeleft"]))
```

Comment:
only if..?

BTW, the induced latency is only about half a second per job (excluding
the time to import subprocess).


On Wed, Aug 19, 2015 at 05:52:23AM -0700, musella wrote:
> > @@ -224,6 +225,13 @@ def parse(self):
> >              self.readProcessIdMap(self.options.processIdMap)
> >          
> >          if self.useAAA:
> > +            import subprocess
> > +            tleft = int(subprocess.check_output(["voms-proxy-info", "--actimeleft"]))
> 
> This code is normally run by cmsRun when it loads the configuration and it may be better not to do it for each job.
> 
> I would probably execute the command only if the 
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-analysis/flashgg/pull/329/files#r37410146